internal/tiltfile/tiltfile_state.go

```diff
@@ -304,6 +304,7 @@ const (
 
 	// local resource functions
 	localResourceN = "local_resource"
+	testN          = "test" // test is just a fork of local resource
```

Comment:
maybe move the comment to line 482 and add "controlled, by `fn.Name()`" or so, to help the reader understand the duplicated "localResource" isn't just bad copy/paste